sql/testdata/default

```diff
@@ -0,0 +1,72 @@
+statement error incompatible column type and default expression: INT vs bool
+CREATE TABLE t (a INT PRIMARY KEY DEFAULT false)
+
+statement ok
+CREATE TABLE t (
+  a INT PRIMARY KEY DEFAULT 42,
+  b TIMESTAMP DEFAULT now(),
+  c FLOAT DEFAULT random()
+)
+
+query TTTT colnames
+SHOW COLUMNS FROM t
+----
+Field Type      Null Default
+a     INT       true 42
+b     TIMESTAMP true now()
+c     FLOAT     true random()
+
+statement ok
+INSERT INTO t VALUES (DEFAULT, DEFAULT, DEFAULT)
+
+query IBB
+SELECT a, b >= now(), c >= 0.0 FROM t
+----
+42 true true
+
+statement ok
+TRUNCATE TABLE t
+
+statement ok
+INSERT INTO t DEFAULT VALUES
+
+query IBB
+SELECT a, b >= now(), c >= 0.0 FROM t
+----
+42 true true
```

Comment:
You probably missed the `TRUNCATE TABLE`. Necessary or we wouldn't be able to insert a row with `a = 42` again.